example/src/components/SingleProductReview/styles.tsx

```diff
@@ -14,6 +14,13 @@ export const LikeHandsActionContainer = styled.View`
   flex-direction: row;
 `
 
+export const LogoWrapper = styled.View`
+  borderRadius: 8;
+  width: 80;
+  marginLeft: auto;
+  marginRight: auto;
+`
```

Comment:
Is it normal to delete the rest of the styles?

Reply:
i just realized that i erased the styles because they throwed errors, but it happened because i added properties like this: overflow: 'hidden', it should be without the ' '